src/main/java/uk/gov/hmcts/cmc/claimstore/ClaimStoreApplication.java

```diff
@@ -3,11 +3,14 @@
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 import org.springframework.boot.autoconfigure.jdbc.DataSourceAutoConfiguration;
+import org.springframework.cloud.client.discovery.EnableDiscoveryClient;
 import org.springframework.cloud.netflix.feign.EnableFeignClients;
 
-@SpringBootApplication(exclude = DataSourceAutoConfiguration.class)
+@SpringBootApplication(exclude = DataSourceAutoConfiguration.class,
+    scanBasePackages = {"uk.gov.hmcts.cmc.claimstore", "uk.gov.hmcts.document"})
 @SuppressWarnings({"HideUtilityClassConstructor", "squid:S1118"}) // Spring needs a constructor, its not a utility class
-@EnableFeignClients
+@EnableDiscoveryClient
+@EnableFeignClients(basePackages = {"uk.gov.hmcts.cmc.claimstore", "uk.gov.hmcts.document"})
```

Comment:
can you reuse the BASE_PACKAGE_NAME constant down below?

Reply:
no